common/src/main/java/com/tencent/rss/common/util/RssUtils.java

```diff
@@ -135,4 +141,55 @@ public static Roaring64NavigableMap deserializeBitMap(byte[] bytes) throws IOExc
     bitmap.deserialize(dataInputStream);
     return bitmap;
   }
+
+  public static List<ShuffleDataSegment> transIndexDataToSegments(
+      ShuffleIndexResult shuffleIndexResult, int readBufferSize) {
+    if (shuffleIndexResult == null || shuffleIndexResult.isEmpty()) {
+      return Lists.newLinkedList();
+    }
+
+    byte[] indexData = shuffleIndexResult.getIndexData();
```

Comment:
if (indexData == null || indexData.isEmpty()) {
  return Lists.newLinkedList();
}

Reply:
Not necessary, if shuffleIndexResult is empty it will return empty list above.

Reply:
OK